metadata.yml

```diff
@@ -0,0 +1,62 @@
+rules:
+- apiGroups: ["*"]
+  apiVersions: ["*"]
+  resources: ["*"]
+  operations: ["CREATE"]
+mutating: false
+contextAware: false
```

Comment:
:D

Reply:
I wonder if this will make things choke out before we have the feature out.

Reply:
Are you referring to the background scan feature, because this policy targets `*` and `CREATE`?

Reply:
I completely misread; I thought we were setting `backgroundAudit` already :man_facepalming:.
And that it would clash with https://github.com/kubewarden/policy-evaluator/blob/main/src/policy_metadata.rs#L123.